css/src/new-dashboard.css

```diff
@@ -1,4 +1,92 @@
 body.toplevel_page_wpseo_dashboard {
+	.notice, .yoast-migrated-notice {
+		display:none;
+	}
+
+	.yoast-new-dashboard-notices:last-child{
+		margin-bottom: 2rem;
+	}
+
+	.yoast-dashboard-notice, .yoast-webinar-dashboard{
+		background: white;
+		border: 1px solid #E0B3CC;
+		box-shadow: 0px 4px 6px -1px #0000001A;
+		padding: 1rem;
+		border-radius: 0.375rem;
+	}
+
+	.yoast-dashboard-notice .yoast-icon {
+		background-color: var(--yoast-color-primary);
+		mask-image: var(--yoast-svg-icon-yoast);
+		-webkit-mask-image: var(--yoast-svg-icon-yoast);
+		mask-size: 100% 100%;
+		-webkit-mask-size: 100% 100%;
+		display: inline-block;
+		width: 17px;
+		height: 17px;
+		margin-right: 8px;
+	}
+
+	.yoast-dashboard-notice .notice-dismiss {
+		position: relative;
+		color: #94A3B8;
+	}
+
+	.yoast-dashboard-notice .notice-dismiss:before{
+		display: none;
+	}
+
+	.yoast-dashboard-notice .notice-yoast img{
+		display: none;
+	}
```

Comment:
Nvm:
![image](https://github.com/user-attachments/assets/4c849cce-5281-4775-b39e-52355b690deb)

Reply:
They needed an extra display none because they were not assigned the new .yoast-webinar-dashboard-image class.

That being said, it's not an extra one, now that I removed the former one.